sdcm/cluster.py

```diff
@@ -2345,6 +2345,46 @@ def write_node_private_ip_file(self):
             private_ip_file.write("%s" % "\n".join(self.get_node_private_ips()))
             private_ip_file.write("\n")
 
+    def populate_db(self, node):
+        raise NotImplementedError
+
+    def stop_db(self, node):
+        raise NotImplementedError
+
+    def start_db(self, node):
+        raise NotImplementedError
+
+    def populate_db_with_ebs(self, ebs_ids):
+
+        # Shutdown all nodes
+        for node in self.nodes:
+            self.stop_db(node)
+
+        # for each node pick up an EBS and attach it
+        for node in self.nodes:
+            # attach the volume
+            ebs_id = ebs_ids.pop()
+            device = "/dev/xvdz"
+            node._instance.attach_volume(VolumeId=ebs_id, Device=device)
+
+            while not node.file_exists(device):
```

Comment:
We can't assume that the device can be attached correctly 100%, it would enter dead loop in error case.

let's use `wait.wait_for(node.file_exists, steps=xx, text='xxx', device=device)`